### PR TITLE
feat: initial application layout

### PR DESCRIPTION
**Pull request by NektariosFifes**
**Description**
The following components without functionality have been added.
- Titlebar
- Sidebar
- Text editor
- container for visualization 

Also added the app icon that was missing.
Also updated certain typescript definitions.

**Related issue(s)**
progresses #88

### CODE REVIEW

**Comment by NektariosFifes** [comment]
@jonaslagoni should i split it in a smaller commit?

File: Desktop/src/renderer/containers/Workbench/components/test.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+const { getArrayMetadata } = require('./test/index');
+
+getArrayMetadata();
```
Comment (jonaslagoni) [inline Desktop/src/renderer/containers/Workbench/components/test.js:3]: Why is this file here?

File: Desktop/src/renderer/containers/Workbench/components/test.d.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export type ArrayMetadata = {
+  length: number;
+  firstObject: any | undefined;
+};
+export function getArrayMetadata(arr: any[]): ArrayMetadata;
```
Comment (jonaslagoni) [inline Desktop/src/renderer/containers/Workbench/components/test.d.ts:5]: Why is this file here?

**Review by jonaslagoni — CHANGES_REQUESTED** [review]
Going forward don't put everything in one PR, I would rather have 100 small PRs than one large 😄 There is no way I can help you out properly as there are too many changes to keep track of. Up to you if you want to split it up.

You have a lot of disabled eslint lines. They are there for a reason, among other reasons to create consistency. Some of them can be a bit of a pain, and in some cases, they might not be needed, but I can't seem to figure out why all of them are needed here? 

Each time you need to disable something, try and second guess why you need to disable them. You can always suggest changing the eslint config, but remember to have your arguments in order why a consistency is not needed, etc 😄

File: Desktop/src/renderer/containers/Workbench/components/test.d.ts (hunk 2 above)
Comment (NektariosFifes) [reply Desktop/src/renderer/containers/Workbench/components/test.d.ts:5]: removed it

**Comment by NektariosFifes** [comment]
i will remove eslints

**Review by jonaslagoni — APPROVED** [review]

**Pull request merged**